windows-installer/Makefile

```diff
@@ -0,0 +1,12 @@
+
+openssl-installer: openssl.nsi
+	makensis.exe /DVERSION=testversion /DBUILD32=_build32 /DBUILD64=_build64 .\openssl.nsi
+
+signed-openssl-installer: openssl.nsi
+	makecert.exe /n "CN=TestCompany" /r /h 0 /eku "1.3.6.1.5.5.7.3.3,1.3.6.1.4.1.311.10.3.13" /sv testcert.pvk testcert.cer
+	pvk2pfx.exe /pvk testcert.pvk /pi testpass /spc testcert.cer /pfx testcert.pfx /po testpass
+	makensis.exe /DVERSION=testversion /DBUILD32=_build32 /DBUILD64=_build64 /DSIGN=testcert.pfx /DSIGNPASS=testpass .\openssl.nsi
+
+clean:
+	del .\*.exe .\test*.*
+
```

Comment:
why not to update https://github.com/openssl/openssl/blob/master/Configurations/windows-makefile.tmpl?

Reply:
because (unless I'm just not spotting it), that makefile is really tailored to building c objectes and linking them into application binaries/libraraies.  This Makefile is really just here to turn an nsis script into a windows installer and optionally sign it.  I'm not sure how I would direct the windows makefile template to do that.

Reply:
Most of all, https://github.com/openssl/openssl/blob/master/Configurations/windows-makefile.tmpl is a template for *one* build, while this makefile handles *multiple* builds in one go.